WIRELESS_OPERATION.md

```diff
@@ -6,6 +6,10 @@ This guide will cover the web interface available via the Attenuator or Wireless
 
 Please see the [ATTENUATOR_FLASHING](ATTENUATOR_FLASHING.md) guide for details on compiling and/or uploading software to your Wireless Adapter controller.
 
+## Hardware Synchronization
+
+In order to view the state of the pack and control it remotely, the two devices must be physically connected and have been sychnronized via the built-in software. On the ESP32 are 2 on-board LED's which will show the current status. A red LED indicates the device is powered and should be accessible via WiFi. A blue LED indicates the device has successfully synchronized with the Proton Pack.
```

Comment:
No apostrophe on LEDs

Reply:
Yeah, a quick search shows 3 in comments in Attenuator.ino and 1 in the Attenuator's Header.h. But that's it from what I can tell.